pkg/freeze/containerd/pause.go

```diff
@@ -89,27 +98,57 @@ func lookupContainerID(ctx context.Context, conn *grpc.ClientConn, podUID, conta
 		},
 	})
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 
 	if len(pods.Items) == 0 {
-		return "", fmt.Errorf("pod %s not found", podUID)
+		return nil, fmt.Errorf("pod %s not found", podUID)
 	}
 	pod := pods.Items[0]
 
 	ctrs, err := client.ListContainers(ctx, &cri.ListContainersRequest{Filter: &cri.ContainerFilter{
 		PodSandboxId: pod.Id,
-		LabelSelector: map[string]string{
-			"io.kubernetes.container.name": containerName,
-		},
 	}})
 	if err != nil {
-		return "", err
+		return nil, err
+	}
+
+	return ctrs, nil
+}
+
+// Pause performs a pause action on a specific container
```

Comment:
comment is a bit out of sync since this takes a list now, although see next comment